### PR TITLE
Add TakeTitan (TTNC)

### PR DESCRIPTION
**Pull request by blinc-usa**
This PR proposes a new Coin  for TakeTitan (TTNC), a new blockchain built using the Cosmos SDK.

- Symbol: TTNC
- Name: TakeTitan

### CODE REVIEW

File: mtgbp/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        ],
+        "base": "mtgbp",
+        "name": "mtgbp",
+        "display": "mtgbp",
+        "symbol": "MTGBP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mtgbp/images/mtgbp.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mtgbp/images/mtgbp.svg"
+        },
+        "coingecko_id": "ttnc",
```
Comment (JeremyParish69) [inline mtgbp/assetlist.json:22]: This CGID doesn't exist at https://api.coingecko.com/api/v3/coins/list. Please remove

File: qfsone/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "qfs",
```
Comment (JeremyParish69) [inline qfsone/assetlist.json:3]: chain_name must match directory name

File: taketitan/chain.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        "kind": "ping.pub",
+        "url": "https://scan.taketitan.com/taketitan",
+        "tx_page": "https://scan.taketitan.com/taketitan/tx/${txHash}",
+        "account_page": "https://scan.taketitan.com/taketitan/accounts/${accountAddress}"
+      }
+    ],
+    "keywords": [],
+    "images": [
+      {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/taketitan/images/taketitan.png",
```
Comment (JeremyParish69) [inline taketitan/chain.json:73]: Why do the images for qfs, mtgbp, and taketitan all look the same?

Comment (blinc-usa) [reply taketitan/chain.json:73]: @JeremyParish69  Thanks for the feedback. I will make all the changes that you have mentioned, and I forgot to replace the logo with the original ones, so I will change it soon and submit my code again.

File: mtgbp/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "mtgbp",
+    "assets": [
+      {
+        "description": "The native staking and governance token of mtgbp.",
+        "extended_description": "mtgbp is a Layer 1 protocol that allows developers to automatically spin up VM-agnostic, parallelized and interoperable dedicated chains, or “Chainlets,” that provide applications with infinite horizontal scalability. Each Chainlet is a replica of the mtgbp Mainnet, with the same validator set and security model.\n\nmtgbp’s mission is to enable the next 1000 chains in gaming and entertainment as part of the growing mtgbp Multiverse.",
```
Comment (JeremyParish69) [inline mtgbp/assetlist.json:7]: These all have the same extended_description. What is the relationship between these three chains? Why are there 3 of the same thing but with different names?

Comment (blinc-usa) [reply mtgbp/assetlist.json:7]: These are three chains for three different clients who wants their own blockchains. We are just learning the cosmos blockchain and cosmos SDK so we are unfortunately rookies in the system. 

We have set the parameters for our clients, it’s standardized from our side. We are learning as we go.

This is why the three chains have the same descriptions, this has not been modified by us (it’s the cosmos sdk standard description) and we missed to change this.

Thanks for pointing it out, we will change it and we will see if we find other places that needs personalizations in the code. @JeremyParish69

File: qfs/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qfs/images/qfs.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qfs/images/qfs.svg"
+        },
+       
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmoschain-registry/master/qfs/images/qfs.png",
+            "svg": "https://raw.githubusercontent.com/cosmoschain-registry/master/qfs/images/qfs.svg",
+            "theme": {
+              "primary_color_hex": "#000000",
```
Comment (JeremyParish69) [inline qfs/assetlist.json:28]: This doesn't seem to be the correct color

File: qfs/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+              "dark_mode": false
+            }
+          },
+          {
+           
+            "theme": {
```
Comment (JeremyParish69) [inline qfs/assetlist.json:34]: You have a second image object with only theme data but no images uploaded

File: qfs/chain.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    "website": "https://qfs.com/",
+    "pretty_name": "qfs",
+    "chain_id": "qfs-1",
+    "bech32_prefix": "qfs",
+    "daemon_name": "qfs",
+    "node_home": "$HOME/.ssc",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip173": "qfs",
```
Comment (JeremyParish69) [inline qfs/chain.json:15]: This is not an accepted property (the same data is provided as bech32_prefix)

File: qfs/chain.json [hunk 8]
```diff
@@ -0,0 +1,77 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "qfs",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://qfs.com/",
+    "pretty_name": "qfs",
```
Comment (JeremyParish69) [inline qfs/chain.json:7]: Why not call it Quantum Financial Systems?

File: taketitan/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+        "denom_units": [
+          {
+            "denom": "ttnc",
+            "exponent": 0
+          }
+        ],
+        "base": "taketitan",
```
Comment (JeremyParish69) [inline taketitan/assetlist.json:14]: this base isn't defined as a denom_unit. The only unit you have defined is "ttnc". Base value must be a defined denom unit

File: taketitan/assetlist.json [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/taketitan/images/taketitan.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/taketitan/images/taketitan.svg",
+            "theme": {
+              "primary_color_hex": "#000000",
+              "dark_mode": false
+            }
+          },
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/taketitan/images/taketitan_white.png",
```
Comment (JeremyParish69) [inline taketitan/assetlist.json:33]: These images are not uploaded.

File: taketitan/chain.json [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+    "pretty_name": "TakeTitan",
+    "chain_id": "taketitan-12",
+    "bech32_prefix": "taketitan",
+    "daemon_name": "ttnc",
+    "node_home": "$HOME/.ssc",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip173": "ttnc",
+    "slip44": 1179993421,
```
Comment (JeremyParish69) [inline taketitan/chain.json:16]: Pending acceptance of SLIP PR (which didn't follow alphabetical order)

File: taketitan/assetlist.json (hunk 9 above)
Comment (blinc-usa) [reply taketitan/assetlist.json:14]: Okay, that means I have to place the same value of base as the demon_unit value "ttnc".

File: qfs/chain.json (hunk 8 above)
Comment (blinc-usa) [reply qfs/chain.json:7]: Okay I will change the "pretty_name" as Quantum Financial Systems .
@JeremyParish69

File: qfs/chain.json (hunk 7 above)
Comment (blinc-usa) [reply qfs/chain.json:15]: @JeremyParish69 To understand why slip173 "qfs" and bech32_prefix are not the same, I need your assistance.

Comment (JeremyParish69) [reply qfs/chain.json]: they pretty much are the same, but SLIP173 isn't defined in the chain.json schema. Just use bech32_prefix

File: mtgbp/chain.json [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+    "pretty_name": "mtgbp",
+    "chain_id": "mtgbp-1",
+    "bech32_prefix": "mtgbp",
+    "daemon_name": "mtgbp",
+    "node_home": "$HOME/.ssc",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip173": "mtgbp",
+    "slip44": 1179993441,
```
Comment (JeremyParish69) [inline mtgbp/chain.json:16]: Pending PR (which need fixing)
https://github.com/satoshilabs/slips/pull/1740

File: qfs/chain.json (hunk 7 above)
Comment (blinc-usa) [reply qfs/chain.json]: @JeremyParish69  Thanks for the help . I will remove SLIP173 from the entire chain.json schema.

File: mtgbp/chain.json (hunk 12 above)
Comment (blinc-usa) [reply mtgbp/chain.json:16]: @JeremyParish69, Can you help me figure out what they are asking me to change in SLIP44 ?
The comments of the reviewer are mentioned below.
    1. Keep the list sorted
    2. Make sure the dec values match the hex values
    3. Please fix also TTNC value, which is wrong

Comment (JeremyParish69) [reply mtgbp/chain.json]: The slip173 entries you've just added at the bottom of the chart, as opposed to arranging it in alphabetical order.
The hexadecimal values in your slip44 entries do not match the decimal values.
Not sure about TTNC

Comment (blinc-usa) [reply mtgbp/chain.json]: @JeremyParish69  Thank you for giving me proper understanding.

Comment (JeremyParish69) [reply mtgbp/chain.json]: I see you made changes in that PR recently but it's still not fixed.

Comment (blinc-usa) [reply mtgbp/chain.json]: Still something is remaining from my side to correct ? @JeremyParish69

Comment (JeremyParish69) [reply mtgbp/chain.json]: Yes, your registration to slip173 and slip44 need to be accepted. You have to match the existing convension.

File: qfs/chain.json (hunk 8 above)
Comment (JeremyParish69) [reply qfs/chain.json]: I don't see that you've made this change yet

File: mtgbp/chain.json (hunk 12 above)
Comment (blinc-usa) [reply mtgbp/chain.json]: Can you share with me those conventions to match?

Comment (JeremyParish69) [reply mtgbp/chain.json]: A maintainer of that repo left on comment on your PR that you haven't addressed. The one that stick out to me is the ordering. You aren't following ascending and alphabetical ording

Comment (blinc-usa) [reply mtgbp/chain.json]: Please check. I have arranged the list in order form.

Comment (blinc-usa) [reply mtgbp/chain.json]: @JeremyParish69 Please check the SLIP0044 and SLIP0172 to get **MERGED**.

File: qfs/assetlist.json [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+      "name": "qfs",
+      "display": "qfs",
+      "symbol": "QFS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qfs/images/qfs.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/qfs/images/qfs.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmoschain-registry/master/qfs/images/qfs.png",
```
Comment (JeremyParish69) [inline qfs/assetlist.json:24]: missing slash (/) between cosmos and chain-registry, causing the check to fail

**Comment by blinc-usa** [comment]
@JeremyParish69 Now My tokens (MTGBP, QFS, & TTNC) are available on the Osmosis Zone app ???
If not, then please assist me with things i have in mind for listing my tokens on the Osmosis Zone app.

**Comment by JeremyParish69** [comment]
> @JeremyParish69 Now My tokens (MTGBP, QFS, & TTNC) are available on the Osmosis Zone app ??? If not, then please assist me with things i have in mind for listing my tokens on the Osmosis Zone app.

Not quite. Osmosis has docs for the asset listing process: https://docs.osmosis.zone/overview/integrate/registration
This page gets right to how to add the asset to the appropriate file: https://github.com/osmosis-labs/assetlists?tab=readme-ov-file#how-to-add-assets
Note that another GitHub account with a longer history will need to comment on your PR, since your account is too new.

**Pull request merged**